src/mailer/mailables/mailable.ts

```
@@ -0,0 +1,23 @@
+import { Address } from 'nodemailer/lib/mailer'
+import { EnvVariable, processEnv } from '../../processEnv'
+
+export class Mailable {
+  to: string | string[] | Address | Address[]
+  from?: string | Address = processEnv(EnvVariable.MAIL_FROM_ADDRESS)
+  subject: string
+
```

I think I'm okay having `Mailable` be abstract so we can force methods like a property for `template`
```ts
  abstract get template(): MailTemplate
````

---

Or we can keep it non-abstract and throw an error if `template` class property is not set when calling `send`.